src/sjur_klemetsen_ex/pa02/snakes_simulations.py

```diff
@@ -0,0 +1,256 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Sjur Spjeld Klemetsen, Ola Flesche Hellenes'
+__email__ = 'sjkl@nmbu.no, olhellen@nmbu.no'
+
+import random as rd
+
+
+class Board:
+    def __init__(self, *args, **kwargs):
+        self.board = args
+        self.ladder = kwargs
+        self.snakes = kwargs
+        if len(args) == 0:
+            self.board = [x for x in range(1, 91)]
+        else:
+            self.board = args
+        if len(kwargs) == 0:
+            self.ladder = {1: 40, 8: 10, 36: 52, 43: 62, 49: 79, 65: 82,
+                           68: 85}
+            self.snakes = {24: 5, 33: 3, 42: 30, 56: 37, 64: 27, 74: 12,
+                           87: 70}
+
+    def goal_reached(self, position):
+        """
+        :return: True or False depending if you have reached the goal or not
+        """
+        if position >= len(self.board):
+            return True
+        else:
+            return False
+
+    def position_adjustment(self, position):
+        """
+        :param position: Position on the board
+        :return: The number of positions you skip if you land on a certain
+        snake or ladder. If position is not on any ladder or snake
+        it returns 0.
+        """
+        for (key1, value1), (key2, value2) in zip(self.ladder.items(),
+                                                  self.snakes.items()):
+            if position == key1:
+                return value1 - key1
+            elif position == key2:
+                return value2 - key2
+            else:
+                continue
+        return 0
+
+
+class Player:
+    """
+    Manages information about a players position, including which board the
+    player is on.
+    """
+    def __init__(self, board):
+        self.board = board
+        self.pos = 0
+        self.num_moves = 0
+
+    def move(self):
+        """
+        Moves a player with a dice on the board
+        :return: Doesnt return anything
+        """
+        roll = rd.randint(1, 6)
+        self.pos += roll
+        for (key1, value1), (key2, value2) in zip(Board().ladder.items(),
```

Comment:
You have position_adjustment method that you can use here instead of repeating the code